Automattic/wp-calypso
# WIP - Send some users to the concierge upsell page after plan purchase

#### Changes proposed in this Pull Request

This adds the logic to conditionally show the new concierge session upsell page to a user after they purchase the Blogger, Personal, or Premium plan.
It's currently hidden behind a feature flag which is only enabled on development.
This upsell will be tested against the GSuite upsell.
The initial percentage of users to allocate to the test group might change.

#### Testing instructions

* Put yourself into the `show` test group.
<img width="599" alt="screen shot 2018-12-17 at 4 01 59 pm" src="https://user-images.githubusercontent.com/690843/50123444-8591bb80-0215-11e9-919f-7fac915569e6.png">

* Create a new site (**don't** choose "Sell products or collect payments" for the "What’s the primary goal..." question).
* Add a Premium plan and domain to the cart.
* Purchase the items.
* After purchase, confirm you're shown the Concierge upsell page.
`[screenshot]`

* Click the button: "Reserve a call for $99.00". You should be taken to the checkout.
* Purchase the session
* Confirm you're taken to the checklist.
`[screenshot]`
